lib/haml_lint/tree/tag_node.rb

```diff
@@ -198,5 +206,17 @@ def script
     def text
       (@value[:value] if @value[:parse]) || ''
     end
+
+    private
+
+    def parsed_attributes
+      HamlLint::RubyParser.new.parse(hash_attributes_source)
```

Comment:
Hi @sds, I spiked on creating a mixin but it felt a little forced. It's less code to reuse the class here, and it's easier to test the parser object as a class vs module. 

What do you think about keeping this line as is? I think that's the only thing blocking this PR.